### PR TITLE
player and cast captions sync

### PR DESCRIPTION
**Pull request by tomasz-rekawek**

### CODE REVIEW

**Review by zemlanin — APPROVED** [review]

File: src/js/tech/ChromecastTech.js [hunk 1]
```diff
@@ -307,6 +308,7 @@ ChromecastTech = {
       this._ui.updateTitle(title);
       this._ui.updateSubtitle(subtitle);
       castSessionObj = castSession.getSessionObj();
+      console.log(castSessionObj, '123121231212312331233');
```
Comment (zemlanin) [inline src/js/tech/ChromecastTech.js:311]: ahem

Comment (tomasz-rekawek) [reply src/js/tech/ChromecastTech.js:311]: How this got here 🤔

File: src/js/tech/ChromecastTech.js [hunk 2]
```diff
@@ -651,7 +653,19 @@ ChromecastTech = {
...
 
+      this._addEventListener(this._remotePlayerController, eventTypes.MEDIA_INFO_CHANGED, () => {
+         if (typeof this._getMediaSession().activeTrackIds[0] === 'number') {
+            const id = this._getMediaSession().activeTrackIds[0];
 
+            const tracks = this._getMediaSession().media.tracks;
+
+            const label = tracks[id].name;
+
+            this._onChangeSubtitleTrack(label);
```
Comment (zemlanin) [inline src/js/tech/ChromecastTech.js:663]: Pass an object (either the whole `track` or just `{ name }`)

Comment (tomasz-rekawek) [reply src/js/tech/ChromecastTech.js:663]: and what do we pass when there are no subtitles 
`{name: 'off'}` ?

**Pull request merged**